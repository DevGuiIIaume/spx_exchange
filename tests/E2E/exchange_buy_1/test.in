2
[T0] BUY 0 Router 1 100;
[T0] BUY 1 Router 2 100;
[T1] BUY 0 Router 1 100;
[T1] BUY 1 Router 2 100;
[T0] DISCONNECT;
[T1] DISCONNECT;
