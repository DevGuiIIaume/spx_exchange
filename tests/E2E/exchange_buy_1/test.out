[SPX] Starting
[SPX] Trading 2 products: GPU Router
[SPX] Created FIFO /tmp/spx_exchange_0
[SPX] Created FIFO /tmp/spx_trader_0
[SPX] Connected to /tmp/spx_exchange_0
[SPX] Connected to /tmp/spx_trader_0
[SPX] Created FIFO /tmp/spx_exchange_1
[SPX] Created FIFO /tmp/spx_trader_1
[SPX] Connected to /tmp/spx_exchange_1
[SPX] Connected to /tmp/spx_trader_1
[SPX] [T0] Parsing command: <BUY 0 Router 1 100>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 1; Sell levels: 0
[SPX]		BUY 1 @ $100 (1 order)
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] [T0] Parsing command: <BUY 1 Router 2 100>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 1; Sell levels: 0
[SPX]		BUY 3 @ $100 (2 orders)
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] [T1] Parsing command: <BUY 0 Router 1 100>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 1; Sell levels: 0
[SPX]		BUY 4 @ $100 (3 orders)
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] [T1] Parsing command: <BUY 1 Router 2 100>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 1; Sell levels: 0
[SPX]		BUY 6 @ $100 (4 orders)
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] Trader 0 disconnected
[SPX] Trader 1 disconnected
[SPX] Trading completed
[SPX] Exchange fees collected: $0
